Docker Build Fails due to TypeScript Error in web/src/main.ts

Hello,

I encountered an issue while trying to build the Docker image for the `whatsapp-contact-sync` project. The build process fails during the `npm run build` step for the web-build with a TypeScript error.

The error message is as follows:

```
src/main.ts(68,41): error TS2345: Argument of type '{ pageTrackerTemplate(to: RouteLocationNormalized): { page_title: string; page_path: string; }; config: { id: string; }; }' is not assignable to parameter of type 'PluginOptions'.
Types of property 'pageTrackerTemplate' are incompatible.
Type '(to: RouteLocationNormalized) => { page_title: string; page_path: string; }' is not assignable to type '() => PageView'. 
```


It appears there's a type mismatch between the `pageTrackerTemplate` function and the `PluginOptions` type. The `pageTrackerTemplate` function seems to be expecting no arguments and a return type of `PageView`, but the provided function takes a `RouteLocationNormalized` argument and returns an object with `page_title` and `page_path`.

--- a/web/src/main.ts
+++ b/web/src/main.ts
@@ -55,14 +55,22 @@ router.beforeEach(
   }
 );
 
+let currentRoute: RouteLocationNormalized;
+
+router.afterEach((to: RouteLocationNormalized) => {
+  // This is run after each navigation, including the initial navigation.
+  currentRoute = to;
+});
+
 const vueTagSettings = {
-  pageTrackerTemplate(to: RouteLocationNormalized) {
+  pageTrackerTemplate: () => {
     return {
-      page_title: to.path,
-      page_path: to.path,
+      page_title: currentRoute.path,
+      page_path: currentRoute.path,
     };
   },
   config: { id: "G-4PJJZRPWG4" },
 };
+
 
 createApp(App).use(router).use(VueGtag, vueTagSettings, router).mount("#app");

